src/main/java/org/spongepowered/api/text/dialogue/Dialogue.java

```diff
@@ -0,0 +1,141 @@
+        return Sponge.getRegistry().createBuilder(Builder.class);
+    }
+
+    /**
+     * Creates a new {@link Instance} for the specified {@link Speaker}(s).
+     *
+     * @param speakers The speakers to send the initial {@link Question} to
+     * @return The newly created {@link Instance}
+     * @throws IllegalArgumentException If {@code speakers} is empty
+     */
+    Instance start(Speaker... speakers);
+
+    interface Builder extends ResettableBuilder<Dialogue, Builder> {
+
+        /**
+         * Sets the first {@link Question} to use when initializing the exchange.
+         *
+         * @param question The question to use
+         * @return This builder, for chaining
+         */
+        Builder initialQuestion(Question question);
+
+        /**
+         * Adds data to populate the initial persistent {@link DataContainer}
+         *
+         * @param query The path to the data
+         * @param input The object to insert
+         * @return This builder, for chaining
+         */
+        Builder initialData(DataQuery query, Object input);
+
+        /**
+         * Adds data to populate the initial persistent {@link DataView}.
+         *
+         * @param view The view to merge with the current one
+         * @return This builder, for chaining
+         */
+        default Builder initialData(DataView view) {
+            view.getValues(true).forEach(this::initialData);
+            return this;
+        }
+
+        /**
+         * Sets whether or not to suppress all messages by default.
+         * {@link Question}s can override this.
+         *
+         * @param suppress Whether or not to suppress messages
+         * @return This builder, for chaining
+         */
+        Builder suppressAllMessages(boolean suppress);
+
+        /**
+         * Sets channels to allow messages from. This is ignored if
+         * {@link #suppressAllMessages(boolean)}, or the {@link Question}
+         * equivalent, is set to true.
+         *
+         * @param channels The channels to allow
+         * @return This builder, for chaining
+         */
+        Builder allowedChannels(Iterable<MessageChannel> channels);
+
+        /**
+         * Sets whether or not output by the {@link Speaker} should be
+         * suppressed by default (as opposed to allowing it to go to, for
+         * example, main chat). This can be overridden by {@link Question}s.
+         *
+         * @param suppress Whether or not to suppress output
+         * @return This builder, for chaining
+         */
+        Builder suppressOutput(boolean suppress);
+
+        /**
+         * Builds the new Dialogue instance.
+         *
+         * @return The new Dialogue
+         */
+        Dialogue build();
+    }
+
+    /**
+     * An instance of a {@link Dialogue} that is in progress.
+     * While {@link Dialogue} is a structure, this is an actual instance of the
+     * back-and-forth that it structures.
+     */
+    interface Instance {
```

Comment:
This should just be merged with Dialogue. and the start method currently there removed.